examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -2742,7 +2742,7 @@
               "side": "server",
               "type": "RegulatoryLocationType",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
Why do you change the storage as well ?

Reply:
I see no changes in the .matter file, so I think this changes client side only (where storage is not used but zap file has some redundancies).